pkg/dependencies/factory.go

```diff
@@ -84,6 +84,7 @@ type Dependencies struct {
 	VSphereValidator          *vsphere.Validator
 	VSphereDefaulter          *vsphere.Defaulter
 	SnowValidator             *snow.AwsClientValidator
+	SnowDefaulters            *snow.Defaulters
```

Comment:
Interesting, why are we adding this field here if it's not being built in this factory?